pages/dkp/kommander/1.2/archives/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: layout.pug
+navigationTitle: Access Documentation Archives
+title: Access Documentation Archives
+menuWeight: 1000
+excerpt: access older versions of Kommander documentation
+enterprise: false
+render: mustache
+---
+# Access archived versions of Kommander documentation
+
+In accordance with our [version policy][policy], we regularly archive older, unsupported versions of our Kommander documentation. At this time, this includes documentation for Kommander 1.1 and below. You can still access older versions of Kommander documentation on our [public github repo][repo], or by locally running a Docker image (experimental):
```

Comment:
This should be a markdown link like:

`[text](link)` i.e. `[version policy](http://url.com/version-policy)`